test/cli/package-export-nested-namespace-conflict/parent/b.rb

```diff
@@ -1,5 +0,0 @@
-# typed: strict
-
-module Parent
```

Comment:
ooc, why delete this test, vs capture the change to the output?

Reply:
The output turned into `No errors! Great job.`, so it seemed not as valuable as it was before. I can add them back in and update the output instead.

Reply:
🙏 thanks! i think it's always value having edge cases like this remain in the codebase just in case.